traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/partneroffering/PartnerOfferingsQuantityFragment.java

```diff
@@ -0,0 +1,141 @@
+package com.guestlogix.traveleruikit.partneroffering;
+
+import android.app.Activity;
+import android.content.Intent;
```

Comment:
Why is this change in this PR? @amirziarati

Reply:
github was making mistakes because there was rebase conflicts. fixed